### PR TITLE
Added Readme

### PR DESCRIPTION
**Pull request by tomin11**
- Added readme
- Removed vl530lx from the package.xml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
-#TODO
+# vl530lx-ROS-driver
+
+## Install directly
```
Comment (bhuvanchandradv) [inline README.md]: this is confusing

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+	
+1. Re-use or create a catkin workspace:
+
+        export CATKIN_WS=~/ws_catkin
+        mkdir -p $CATKIN_WS/src
+        cd $CATKIN_WS/src
+
+1. Download the repository:
+
+        git clone git@github.com:rapyuta-robotics/vl53l0x_ros_driver.git
```
Comment (bhuvanchandradv) [inline README.md]: clone it recursively, since we have submodule

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+1. Configure and build the workspace:
+
+        cd ..
+        catkin config --extend /opt/ros/melodic --cmake-args -DCMAKE_BUILD_TYPE=Release
+        catkin build 
+ 
+
+1. Source the workspace.
+
+        source ./devel/setup.bash
```
Comment (bhuvanchandradv) [inline README.md]: use absolute path

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
...
+
+### Read data from the sensors
+1. Run the launch 
+
+        roslaunch vl53l0x_driver vl53l0x_driver.launch
```
Comment (bhuvanchandradv) [inline README.md:44]: can you also show the sample topic echo of sensor

File: package.xml [hunk 5]
```diff
@@ -14,12 +14,10 @@
   <build_depend>std_msgs</build_depend>
   <build_depend>geometry_msgs</build_depend>
   <build_depend>message_generation</build_depend>
-  <build_depend>vl53l0x</build_depend>
```
Comment (bhuvanchandradv) [inline package.xml:17]: comment it out instead of removing

File: README.md [hunk 6]
```diff
@@ -1 +1,39 @@
-#TODO
+# vl530lx-ROS-driver
+
+## Install directly
+
+### Build from Source
```
Comment (bhuvanchandradv) [inline README.md:3]: add libsoc build dependency

File: README.md (hunk 1 above)
Comment (tomin11) [reply README.md]: Ok

File: README.md (hunk 2 above)
Comment (tomin11) [reply README.md]: ok

File: README.md (hunk 6 above)
Comment (tomin11) [reply README.md:3]: ok

File: README.md (hunk 3 above)
Comment (tomin11) [reply README.md]: ok

File: package.xml (hunk 5 above)
Comment (tomin11) [reply package.xml:17]: ok

Comment (tomin11) [reply package.xml:17]: need to check this

File: README.md (hunk 4 above)
Comment (tomin11) [reply README.md:44]: ok

**Review by bhuvanchandradv — APPROVED** [review]

**Pull request merged**